core/jobs/batch_jobs/blog_validation_jobs.py

```diff
@@ -138,7 +138,7 @@ def expand(
             | 'Discard %s key' % self.property_name >> (
                 beam.Values()) # pylint: disable=no-value-for-parameter
             | 'Discard models with unique %s' % self.property_name >> (
-                beam.Filter(lambda models: len(models) > 1))
+                beam.Filter(lambda models: len(list(models)) > 1))
```

Comment:
Why is this changed?

Reply:
Running the job gave the following error: `object of type '_UnwindowedValues' has no len()` , 
This is because - the Direct Runner returned a list was an inconsistency that was fixed in Beam sometime ago. 
Now it return an iterable which needs to be converetd into a list!
https://stackoverflow.com/questions/42276520/what-does-object-of-type-unwindowedvalues-has-no-len-mean

Reply:
Okay, great.